autopush/tests/test_main.py

```diff
@@ -235,10 +235,14 @@ def test_ssl(self):
             "--s3_bucket=none",
         ])
 
-    def test_bad_senderidlist(self):
+    @patch('autopush.main._parse_endpoint')
```

Comment:
Huh, was getting local coverage misses off of this. My local install was using values pulled from local files, which is horribly invalid anyway. This ensures that log.critical is called.